docssrc/src/argument_list.md

```diff
@@ -57,6 +57,20 @@ $$\downarrow$$
 
 $$\downarrow$$
 
+> ### Allowing any value (Optional)
```

Comment:
In this section I think we should mention that the `allowAnyValue` option only works for Strings.